docs/contracts/index.md

```diff
@@ -48,10 +48,11 @@ AssertionError: divide argument '2' did not match type annotation for parameter
 
 ## API
 
-The `python_ta.contracts` module offers two functions for enabling contract checking.
+The `python_ta.contracts` module offers three functions for enabling contract checking.
```

Comment:
You're on the right track here, but the documentation could be improved further. The new function `check_invariants` isn't used to *enable* contract checking, but rather to manually check the representation invariants of an object.

You can keep the generated docstring where it is, but move the intro blurb ("The third, `check_invariants`, ...") to directly above it, mimicking the style of how the constants `ENABLE_CONTRACT_CHECKING` etc. are introduced.